app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    instance_list = [Person(human["name"], human["age"]) for human in people]
+
+    for human in people:
+        human_name = human["name"]
+        if human.get("wife") is not None:
```

Comment:
you can make it easier:
`if human.get("wife") is not None:` <=> `if human.get("wife"):`
that is the same but looks better

Reply:
still not fixed. The same about "husband" key